Sources/TootSDK/HTTP/TimelineLocality.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//
+//  Created by Dale Price on 3/1/23.
+//
+
+import Foundation
+
+/// Specifies whether a timeline request should be restricted to local or remote statuses
+public enum TimelineLocality: Codable, Sendable {
```

Comment:
The notion of "locality" if I understand what you mean by it is related to a Post, not to a timeline (is the post local or is the post remote). There are a number of timelines that do not have a clear indication if they include only local or remote posts or both, and so we'd need "unknown/undefined" as a case to the enum which feels confusing.

Let's remove the notion of `locality` in favour of additional filter fields in the query structs of each timeline, and only for the timelines that have support for them (like `only_media`).

Reply:
It referred to where a timeline endpoint should filter posts from, but if it isn’t clear then it’s probably better to stick to separate `local` and `remote` booleans like Mastodon has. Done in 0b1ff77